packages/console/src/types/guide.ts

```diff
@@ -1,5 +1,4 @@
 export type GuideForm = {
   redirectUris: string[];
   postLogoutRedirectUris: string[];
-  connectorConfigJson: string;
```

Comment:
It seems that we should not delete this prop; for it's used in the connector details page.

Reply:
No, the connector details page doesn't use this type.